js/core/src/Embed.svelte

```diff
@@ -24,32 +25,73 @@
 	style:flex-grow={!display ? "1" : "auto"}
 	data-iframe-height
 >
-	<div class="main">
-		<slot />
-	</div>
-	{#if display && space && info}
-		<div class="info">
-			<span>
-				<a href="https://huggingface.co/spaces/{space}" class="title">{space}</a
-				>
-			</span>
-			<span>
-				{$_("common.built_with")}
-				<a class="gradio" href="https://gradio.app">Gradio</a>.
-			</span>
-			<span>
-				{$_("common.hosted_on")}
-				<a class="hf" href="https://huggingface.co/spaces"
-					><span class="space-logo">
-						<img src={space_logo} alt="Hugging Face Space" />
-					</span> Spaces</a
-				>
-			</span>
+	{#if pages.length > 1}
+		<div class="nav-holder">
+			<nav class="fillable" class:fill_width>
```

Comment:
There is a bug here where the `Home` button will always point to `/` but that's not necessarily where the gradio app is hosted. Could be a different path like in `gr.mount_gradio_app`